lib/ecl/ecl_unsmry_loader.cpp

```diff
@@ -65,6 +65,10 @@ std::vector<double> unsmry_loader::get_vector(int pos) const {
     data[index] = *data_value;
   }
   int_vector_free( index_map );
+
+  if (ecl_file_view_flags_set(file_view, ECL_FILE_CLOSE_STREAM))
+      ecl_file_view_fclose_stream(file_view);
```

Comment:
Please adhere to the 2 space indent of the rest of the file

Reply:
Perhaps the stream should be closed at the end of the other publicly exposed functions here as well?

Reply:
I did some tests with adding the closing statement to other functions as well, but this resulted in a huge performance penalty (see unsmry_loader::iget). This PR is sufficient to fix the issues in ResInsight.